components/core/src/glt/Utils.cpp

```diff
@@ -84,57 +84,6 @@ ErrorCode create_directory_structure(string const& path, mode_t mode) {
     return ErrorCode_Success;
 }
 
-string get_parent_directory_path(string const& path) {
```

Comment:
Technically we don't have to remove those under src/glt/ since it is a different binary (and quite out of dated). but I guess it doesn't harm.